examples/grafana/README.md

```diff
@@ -57,11 +60,11 @@ If you are running a cluster of Fn servers then you can specify them all here.
 
 Now start Prometheus, specifying this config file:
 ```
-docker run --name=prometheus -d -p 9090:9090 \
-  --mount type=bind,source=`pwd`/prometheus.yml,target=/etc/prometheus/prometheus.yml \
-  --add-host="localhost:`route | grep default | awk '{print $2}'`" prom/prometheus
+  docker run --name=prometheus -d -p 9090:9090 \
+    --mount type=bind,source=`pwd`/prometheus.yml,target=/etc/prometheus/prometheus.yml \
```

Comment:
can't `--mount` can be replaced with volume attached to `/etc/prometheus`:

```
-v `pwd`:/etc/prometheus
```

Reply:
also in prometheus.yml, need to use docker.for.mac.localhost instead of localhost.

```
    static_configs:
      # Specify all the fn servers from which metrics will be scraped
      - targets: ['docker.for.mac.localhost:8080'] # Uses /metrics by default
```

Reply:
Better to use docket dns names instead. The thing you suggest is very dind-specific. We can rely on dockets flat network.

Reply:
@denismakogon ok how? it didn't work out of the box for me, which means it probably won't work for others out of the box.

Reply:
Since the instructions have only been discovered to not work on Mac this PR will need to be completed by someone with the required device.